WcaOnRails/app/models/country.rb

```diff
@@ -5,6 +5,24 @@ class Country < ApplicationRecord
   WCA_STATES_JSON_PATH = Rails.root.to_s + "/config/wca-states.json"
   self.table_name = "Countries"
 
+  ALL_TIMEZONES_MAPPING = begin
+                            all_tz = ActiveSupport::TimeZone::MAPPING
+                            grouped_tz = all_tz.group_by { |k, v| v }
+                            duplicates = grouped_tz.select { |k, v| v.size > 1 }
+                            duplicates.each do |tz_id, tz_entries|
+                              selected_name = tz_id
```

Comment:
I spotted a subtle bug:

![image](https://user-images.githubusercontent.com/17034772/40633028-f75167f2-62ec-11e8-922b-3fdd4a3b3d3c.png)

There are two Mexico City entries:
```ruby
"Guadalajara" => "America/Mexico_City",
"Mexico City" => "America/Mexico_City"
```
As the `selected_name` is initially `tz_id` and the `.include?` below doesn't match for any of these, so `tz_id` stays as the name.

Reply:
Haha, that's not a bug but where "being smart" from the line below stops :p
I've updated the `include?` below to use a name where spaces are replaced by `_` which makes the matching process a bit smarter :)